notebooks/IONotebook.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
Thanks for the notebook! Here some improvements:
- Include the url in `graphio.Format` (i.e., [`graphio.Format`](https://networkit.github.io/dev-docs/python_api/graphio.html?highlight=format#networkit.graphio.Format))
- Do not use C++ syntax in Python. For example, instead of `SNAPGraphReader(bool directed = false, const bool& remapNodes = true, const count& nodeCount = 0)`  use the Python syntax `SNAPGraphReader(directed = False, remapNodes = True, nodeCount = 0)`.
- For all readers/writers, it would be nice to have a link to the documentation every time we mention the them in the text
- Add a link to [Thrill](http://project-thrill.org/) when you mention it

Reply:
Links to the documentation are a good idea. However, I would suggest to only add them whenever a reader/writer is mentioned first.

Reply:
I have addressed the comments, and amended the specific commit.

Reply:
Thank you for addressing the comments, just a few remarks:
- You are still using the C++ syntax in the SNAPGraphReader, use the Python syntax instead
- Same thing for the EdgeListReader, KONECTGraphReader, GraphToolBinaryWriter, ThrillBinaryReader, and NetworkitBinaryWriter
- Remove the parentheses after each separator of the EdgeListReaders, e.g., EdgeListSpaceZero() -> EdgeListSpaceZero

Reply:
I have pushed the changes; I hope I haven't overlooked anything this time.